### ISSUE TITLE
Loading housing prices without internet

### ISSUE THREAD

**Issue by WaetzoldP** [ageron/handson-ml#247]
Hi,
the internet of my company does not let me download the housing prices. I've discovered the data locally, so is it possible to load the data from there instead of downloading then from the server?
Regards,
Wätzold

**Comment by WaetzoldP** [comment]
I've found a Solution:```

​#housing = fetch_california_housing()
housing = fetch_california_housing(data_home=".../housing/",
                                                    download_if_missing=False)
```
amd place the file "cal_housing_py3.pkz", which you can download from the internet.

**Comment by ageron** [comment]
Hi @WaetzoldP ,
If you are referring to the `sklearn.datasets.fetch_california_housing()` function, it will load the original housing dataset.  The dataset I use in the book is based on it, but I tweaked it a bit to make it a bit more challenging.  I added a categorical column (`ocean_proximity`) and I removed about 300 values in the `total_bedrooms` column.
I was going to answer, but I see you found the solution while I was typing this!  :)
Cheers,
Aurélien
